### PR TITLE
Move recent changes from Beta to Main

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #31, #32, #34 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Move recent changes from the Beta branch to the Main branch, including a new function for custom name registration and a refactor of color styles to use tier 3 tokens. Update the README to reflect the new version of the auro-loader script.

New Features:
- Introduce a function to handle custom name registration for the auro-loader component.

Enhancements:
- Refactor color styles to use tier 3 tokens, improving the styling consistency and flexibility of the auro-loader component.

Documentation:
- Update README to reflect the new version of the auro-loader script from 2.0.6 to 2.1.0-beta.2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ In cases where the project is not able to process JS assets, there are pre-proce
 ```html
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@4.3.1/dist/tokens/CSSCustomProperties.css" />
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@5.0.8/dist/bundled/essentials.css" />
-<script src="https://cdn.jsdelivr.net/npm/@aurodesignsystem/auro-loader@2.0.6/dist/auro-loader__bundled.js" type="module"></script>
+<script src="https://cdn.jsdelivr.net/npm/@aurodesignsystem/auro-loader@2.1.0-beta.2/dist/auro-loader__bundled.js" type="module"></script>
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply README.md]: Not addressing this suggestion at this time as it is not related to the work in the PR.

File: CHANGELOG.md
Comment (jason-capsule42) [inline CHANGELOG.md:1]: Remove beta release update

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (jordanjones243) [reply CHANGELOG.md:1]: It was already removed...

**Review by DukeFerdinand — APPROVED** [review]

**Pull request merged**